irmaclient/sdjwtvc_storage.go

```diff
@@ -453,11 +441,11 @@ func (s *InMemorySdJwtVcStorage) RemoveCredentialByHash(hash string) ([]jwk.Key,
 	return nil, nil
 }
 
-func (s *InMemorySdJwtVcStorage) GetCredentialInfoList() irma.CredentialInfoList {
-	result := irma.CredentialInfoList{}
+func (s *InMemorySdJwtVcStorage) GetCredentialInfoList() []SdJwtVcBatchMetadata {
```

Comment:
Maybe add a comment as to why the function is named like this, because it might become kind of confusing later.